ios/Event+JSON.swift

```diff
@@ -131,6 +131,38 @@ extension PlayEvent: TimedEventType {}
 extension PausedEvent: TimedEventType {}
 extension PlayingEvent: TimedEventType {}
 
+extension AudioAddedEvent {
+    func toJSON() -> [AnyHashable: Any] {
+        [
+            "name": name,
+            "timestamp": timestamp,
+            "audioTrack": RCTConvert.audioTrackJson(audioTrack)
```

Comment:
Property `time` is missing here

```suggestion
            "audioTrack": RCTConvert.audioTrackJson(audioTrack),
            "time": time
```